draft-zip.md

```diff
@@ -80,12 +82,17 @@ The coinbase transaction at block height `height` MUST have a `zsfDeposit` that
 TODO ZIP Editors:
 - work out how to deal with pre-v6 transactions which don't have the zsfDeposit() field. For example, by requiring the remaining value in the transparent transaction value pool of a coinbase transaction to be greater than or equal to 60% of the fee
 - consider imposing this requirement on v6 transactions instead of an explicit `ZSF_DEPOSIT` field requirement
+- work out the necessary protocol spec changes.
+
+## Deployment
+
+The implementation of this ZIP MUST be deployed at the same time or after the Zcash Sustainability Fund is established.
```

Comment:
Question for the ZIP owners: What order do you want to deploy the last two ZSF ZIPs in?

If the ZSF ZIP deployments are split across multiple network upgrades, do you want subsidy smoothing (and withdrawals from the ZSF) to happen first? Or do you want to put fees into the ZSF before there are any payouts from it?

They might also be deployed at the same time. But we need to know the order in case one of them is delayed or doesn't end up being deployed.

```suggestion
The implementation of this ZIP MUST be deployed at the same time or after the Zcash Sustainability Fund (ZIP-XXX), and subsidy smoothing  (ZIP-YYY).
```

Reply:
I think we'll aim to deploy all 3 ZIPs at the same time. This is technically doable and I don't see much benefit in splitting the deployment into multiple steps.

Reply:
Sure, but just to be clear, the choice of which ZIPs to deploy lies with ECC & ZF, not the ZIP Editors. Our role is to make it clear which ZIPs can be deployed independently if necessary, or what order they need to be deployed in.